src/class-parsely.php

```diff
@@ -1788,4 +1841,26 @@ public function parsely_is_user_logged_in() {
 	public function return_personalized_json() {
 
 	}
+
+	/**
+	 * Convert JSON-LD types to respective Parsely page types
+	 *
+	 * @see https://www.parse.ly/help/integration/metatags#field-description
+	 * @since 2.5.0
+	 *
+	 * @param  $type string
+	 * @return string
+	 */
+	public function convert_jsonld_to_parsely_type( $type ) {
+		if ( in_array( $type, $this->allowed_jsonld_post_types ) ) {
+			return 'post';
+		}
+
+		if ( in_array( $type, $this->allowed_jsonld_non_post_types ) ) {
+			return 'index';
+		}
+
```

Comment:
If the fallback is `index`, then it seems like this conditional could be removed completely.

```suggestion
```